src/tokens.ts

```diff
@@ -103,6 +103,7 @@ export const TOKEN_SYMBOLS_MAP = {
       [CHAIN_IDs.BLAST_SEPOLIA]: "0x4200000000000000000000000000000000000023",
       [CHAIN_IDs.INK]: "0x4200000000000000000000000000000000000006",
       [CHAIN_IDs.INK_SEPOLIA]: "0x4200000000000000000000000000000000000006",
+      [CHAIN_IDs.LENS]: "0xE5ecd226b3032910CEaa43ba92EE8232f8237553",
```

Comment:
nb. It's counter-intuitive to define Lens WETH in the ETH section, but it's necessary due to some constraint in the fe (iirc). I previously bumped into that when removing the ETH definition on Polygon.